java/docker-build/build-in-docker

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Build the spark-rapids-jni artifact in a Docker container with devtoolset
+
+set -e
+
+# Base paths relative to this script's location
+SCRIPTDIR=$(cd "$(dirname "$0")"; pwd)
+
+LOCAL_MAVEN_REPO=${LOCAL_MAVEN_REPO:-"$HOME/.m2/repository"}
+export CMAKE_GENERATOR=${CMAKE_GENERATOR:-"Ninja"}
+export CUDA_VERSION=${CUDA_VERSION:-12.9.1}
+# CUDA_CLASSIFIER=cuda${CUDA_VERSION%%.*}  # Currently unused, but may be needed for future CUDA version handling
+
+# Set env for arm64 build, The possible values of 'uname -m' : [x86_64/i386/aarch64/mips/...]
+if [ "$(uname -m)" == "aarch64" ]; then
+  # ARM-specific settings.
+  echo "Set ARM Specific settings"
+fi
+
+# TODO: Settings to build for different CUDA versions, different GPU archs, CPU archs, etc.
```

Comment:
We should either error out if no arguments are passed into `build-in-docker` or default to `libcuvs java --allgpuarch`